src/pages/PageSearchResult.vue

```diff
@@ -84,7 +84,10 @@ import SearchResultList from '@/components/search/SearchResultList.vue';
 import SearchToggle from '@/components/search/SearchToggle.vue';
 import Fetcher from '@/assets/js/fetcher';
 
-const fetcher = new Fetcher({ failoverOn: [302] });
+const fetcher = new Fetcher({
+  isError: (e) =>
+    e instanceof TypeError && e.message.startsWith('NetworkError'),
```

Comment:
Will search throw errors which aren't NetworkErrors? Will other fetches throw errors which aren't NetworkErrors? If they do, do we want them to reload on all fetch errors, but only search to reload on NetworkErrors?

Reply:
Mostly keeping previous behavior to avoid breaking things for now.